app/services/research_trends_report_generator.rb

```diff
@@ -81,8 +81,8 @@ def generate_totals(years_totals)
     totals_row = ['TOTAL']
     grand_total = 0
     Range.new(start_year, end_year).each do |year|
```

Comment:
Should we cast `start_year` & `end_year` `#to_i`?

Reply:
Concur. Fixed.